README.md

```diff
@@ -273,7 +273,7 @@ pub struct Region {
 
 If you followed the [instructions above](#Creating), you should have a runable
 smart contract. You may notice that all of the Wasm exports are taken care of by
-`lib.rs`, which should shouldn't need to modify. What you need to do is simply
+`lib.rs`, which shouldn't need to modify. What you need to do is simply
```

Comment:
There seems to me a word missing

```suggestion
`lib.rs`, which you shouldn't need to modify. What you need to do is simply
```

Reply:
Ahh yes now it's correct. I'll run the format in a minute